### ISSUE TITLE
Could not execute entrypoint stage 'main' due to errors, provided by 'map_atlases'!

### ISSUE THREAD

**Issue by Maelep** [Pepperoni-Jabroni/MapAtlases#13]
`[19:25:13] [main/FATAL]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'map_atlases'!
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53) ~[fabric-server-launch.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36) ~[fabric-server-launch.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointServer.start(EntrypointServer.java:32) ~[fabric-server-launch.jar:?]
        at net.minecraft.server.Main.main(Main.java:92) [intermediary-server.jar:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_282]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_282]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_282]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_282]
        at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28) [fabric-server-launch.jar:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_282]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_282]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_282]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_282]
        at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:62) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:106) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:50) [fabric-server-launch.jar:?]
Caused by: java.lang.NoClassDefFoundError: me/shedaniel/autoconfig/ConfigData
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_282]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_282]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_282]
        at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:163) ~[fabric-server-launch.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_282]
        at pepjebs.mapatlases.MapAtlasesMod.onInitialize(MapAtlasesMod.java:53) ~[map-atlases-1.2.5.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
        ... 17 more
Caused by: java.lang.ClassNotFoundException: me.shedaniel.autoconfig.ConfigData
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_282]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_282]
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[?:1.8.0_282]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_282]
        at net.fabricmc.loader.launch.server.InjectingURLClassLoader.loadClass(InjectingURLClassLoader.java:56) ~[fabric-server-launch.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_282]
        at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:168) ~[fabric-server-launch.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_282]
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_282]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_282]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_282]
        at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:163) ~[fabric-server-launch.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_282]
        at pepjebs.mapatlases.MapAtlasesMod.onInitialize(MapAtlasesMod.java:53) ~[map-atlases-1.2.5.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
        ... 17 more`

**Comment by Pepperoni-Jabroni** [comment]
Thank you for taking the time to file a bug report, u/Maelep! Can you provide me more information about your mod's config and your list of all installed mods?

**Comment by renandadalte** [comment]
So, I've got this error when I tried to run a plain Fabric install with just Fabric API and Maps Atlases, but I knew that alongside a bunch of mods I was testing it worked, so I started a try an error check, mod by mod. I found out that ShulkerBoxTooltip was managing to keep it working for an unknown reason. When I removed ShulkekBoxTooltip, Stoneholm mod prevented the game to start saying Cloth Config API was missing (and I wasn't even on the pack to get started), so I downloaded it and for my surprise, Cloth Config API not just worked for ShulkerBoxTooltip, but for Map Atlases too. So I created a new plain Fabric instance with just Fabric API, Map Atlases and Cloth Config API and bingo!

I really don't know why, but Map Atlases needs something from Cloth Config API and installing it can fix this error.
